index.js

```diff
@@ -109,7 +110,7 @@ export default class SlotMachine extends Component {
             const newAnimations = paddedStr.split('').map((char, i) => {
                 const index = range.indexOf(char);
                 const animationValue = -1 * (index) * newProps.height;
-                return Animated.timing(this.state.values[i], {toValue: animationValue, duration, easing});
+                return Animated.timing(this.state.values[i], {toValue: animationValue, duration, easing, useNativeDriver: this.props.useNativeDriver});
```

Comment:
take it from newProps, not this.props. Also, you can declare useNativeDriver on line 104, and just use it here.